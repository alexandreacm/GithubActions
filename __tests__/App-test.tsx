// import 'react-native';
import React from 'react';
import App from '../App';

import renderer, {create} from 'react-test-renderer';
import {render} from '@testing-library/react-native';

describe('All tests', () => {
  const tree = create(<App />).toJSON();

  test('renders correctly snapshot test', () => {
    expect(tree).toMatchSnapshot();
  });

  test('renders correctly', () => {
    renderer.create(<App />);
  });

  test('renders correctly App component with @testing-library', () => {
    render(<App />);
  });
});
