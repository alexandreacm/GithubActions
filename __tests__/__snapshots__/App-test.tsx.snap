// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All tests renders correctly snapshot test 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#121212",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#FFF",
        "fontSize": 25,
      }
    }
  >
    Integration tests with
  </Text>
  <Text
    style={
      Object {
        "color": "#FFF",
        "fontSize": 25,
      }
    }
  >
    GithubAction
  </Text>
</View>
`;
